CHANGELOG.md

```diff
@@ -2,17 +2,73 @@
 
 ## Unreleased
 
+## v0.97.0
 
-### 💡 Enhancements 💡
+### 🚀 New components 🚀
 
-- (Splunk) 'memory_ballast` has been removed. If GOMEMLIMIT env var is not set, then 90% of the total available memory limit is set by default. 
+- (Splunk) Add AWS container insights receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
+- (Splunk) Add AWS ECS container metrics receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
+- (Splunk) Add Apache metrics receiver ([#4505](https://github.com/signalfx/splunk-otel-collector/pull/4505))
+
+### 💡 Enhancements 💡
+
+- (Core) `configtls`: Validates TLS min_version and max_version ([#9475](https://github.com/open-telemetry/opentelemetry-collector/issues/9475))
+  Introduces `Validate()` method in TLSSetting.
+- (Contrib) `exporter/loadbalancingexporter`: Adding AWS Cloud Map for service discovery of Collectors backend. ([#27241](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27241))
+- (Contrib) `awss3exporter`: add `compression` option to enable file compression on S3 ([#27872](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27872))
+    Add `compression` option to compress files using `compress/gzip` library before uploading to S3.
+- (Contrib) `awss3exporter`: Add support for encoding extension to awss3exporter ([#30554](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30554))
+- (Contrib) `deltatocumulativeprocessor`: introduce configurable stream limit ([#31488](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31488))
+    Adds `max_streams` option that allows to set upper bound (default = unlimited)
+    to the number of tracked streams. Any additional streams exceeding the limit
+    are dropped.
+- (Contrib)`deltatocumulativeprocessor`: expire stale series ([#30705](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30705), [#31016](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31016))
+  Adds `max_stale` option that allows to set an interval (default = `5min`)
+  after which a series that no longer receives new samples is removed from
+  tracking.
```

Comment:
```suggestion
```
From what I can tell, we have the `cumulativetodeltaprocessor` in our distro, but not the `deltatocumulativeprocessor`.